src/Illuminate/Routing/Console/ControllerMakeCommand.php

```diff
@@ -268,6 +270,35 @@ protected function generateFormRequests($modelClass, $storeRequestClass, $update
         return [$storeRequestClass, $updateRequestClass];
     }
 
+    /**
+     * Build the return type replacement values.
+     *
+     * @param  array  $replace
+     * @param  string|null  $return
+     * @return array
+     */
+    protected function buildReturnTypeReplacements(array $replace, string|null $returnType)
+    {
+        $returnTypes = match ($returnType) {
+            'response' => ['use Illuminate\Http\Response;', ': Response'],
+            'json' => ['use Illuminate\\Http\\JsonResponse;', ': JsonResponse'],
```

Comment:
Typo `\\`

Reply:
@antonkomarev thanks, well spotted! It weirdly worked with both single and double backslash. Fixed now!